iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -68,7 +70,15 @@ public async Task RunSampleAsync(TimeSpan sampleRunningTime)
             try
             {
                 await InitializeAndOpenClientAsync();
-                await Task.WhenAll(SendMessagesAsync(cts.Token), ReceiveMessagesAsync(cts.Token));
+
+                if (_transportType.Equals(TransportType.Http1))

```

Comment:
Why `.Equals()` and not `==`?

Reply:
I tend to associate == with comparison by reference, while .Equals() as comparison by value; so I chose to use .Equals() here

Reply:
Transport type is an enum, effectively an int, so that is irrelevant in this case, right?

I'm pretty sure in C# the default should be == unless you explicitly need `Equals()`.